app/App.js

```diff
@@ -15,11 +16,11 @@ import { ViroVRSceneNavigator, ViroARSceneNavigator } from 'react-viro';
 var sharedProps = {
   apiKey: 'API_KEY_HERE',
 };
-
+// CR * var/pasted code
 // Sets the default scene you want for AR and VR
 var InitialARScene = require('./js/HelloWorldSceneAR');
 var chooseYourBase = require('./js/ARPlaneSelector');
-var chooseAnImageBase = require('./js/ARImageMarker');
+var chooseAnImageBase = require('./js/ARImageMarker'); // CR * capitalize
```

Comment:
The default export of `./js/ARImageMarker` is a React component. I would suggest storing the imported value with a capital letter to make that more clear, like line 21.